client/my-sites/email/email-providers-in-depth-comparison/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { translate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { InDepthComparison, ProviderComparison } from './in-depth-comparison';
+
+type EmailProvidersInDepthComparisonProps = {
+	comparisonContext: string;
+	selectedDomainName: string;
+	siteName: string;
+	source: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+const ProfessionalEmailComparisonObject: ProviderComparison = {
```

Comment:
I believe we shouldn't uppercase the first letter of a variable. We could also remove the suffix, and refine the name of the type so both sides match:

```suggestion
const professionalEmailComparison: EmailProviderComparison = {
```